src/components/FormItem/index.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
```

Comment:
What exactly does this component do? The switch statement is clever but maybe it's better to separate into separate components for readability? What exactly is optionsArr? If you're exposing some internal API make sure to document it ^_^

Reply:
@styvone slap a comment over optionsArr and other non obvious functionality.  Also, add a readme in the folder ```./FormItem/README.md``` and give examples of usage if you really really really really want to lol